environments/kubernetes/observatorium.libsonnet

```diff
@@ -1,10 +1,17 @@
-local kt = (import 'kube-thanos.libsonnet');
-
 (import 'observatorium/observatorium-api.libsonnet') {
   observatorium+:: {
-    namespace:: 'observatorium',
+    local namespace = 'observatorium',
```

Comment:
Why is this local necessary? Vs `namespace:: 'observatorium'`

Reply:
Actually it's not. Just a common pattern that we used in this repo, in case we need it in current config. I just followed it in here as well.

Reply:
If you prefer simpler version, I can fix it?

Reply:
Yes please. I’d prefer we not establish any more unneeded jsonnet patterns I think in some of those other uses we actually used the local variable more than once